data/ecosystems/s/solana.toml

```diff
@@ -8727,3 +8727,5602 @@ url = "https://github.com/ZYJLiu/token-creation-frontend"
+
+[[repo]]
+url = "https://github.com/WillSnowdon/slurp-social"
+
+[[repo]]
+url = "https://github.com/15-Lippo/spx"
+
+[[repo]]
+url = "https://github.com/gcode01/nigeriaclient"
+
+[[repo]]
+url = "https://github.com/withcomet/react"
+
+[[repo]]
+url = "https://github.com/Daniel-Schultz13/mango-client-v3"
+
+[[repo]]
+url = "https://github.com/ubeydtalha/community_token_solana_contract"
+
+[[repo]]
+url = "https://github.com/overoxidize/rustcrow"
+
+[[repo]]
+url = "https://github.com/balexander123/solana-guide-review-comments"
+
+[[repo]]
+url = "https://github.com/cinojosa0705/solana-bpf-program-template"
+
+[[repo]]
+url = "https://github.com/thisisommore/solana_test"
+
+[[repo]]
+url = "https://github.com/JuanMarchetto/paulx-escrow"
+
+[[repo]]
+url = "https://github.com/irffanasiff/solana-escrow-program"
+
+[[repo]]
+url = "https://github.com/Jim97M/locoin"
+
+[[repo]]
+url = "https://github.com/mdb1710/escrow"
+
+[[repo]]
+url = "https://github.com/mahmudsudo/solana--escrow"
+
+[[repo]]
+url = "https://github.com/Vishvesh-rao/solana-test"
+
+[[repo]]
+url = "https://github.com/Shubharma/Shubham-Solana-Q1-2023"
+
+[[repo]]
+url = "https://github.com/Amulet-Protocol/amulet-integration"
+
+[[repo]]
+url = "https://github.com/raghav-rama/todo-solana"
+
+[[repo]]
+url = "https://github.com/kumailnaqvi354/Solana-Todo"
+
+[[repo]]
+url = "https://github.com/gcunruh/tipsea-solana"
+
+[[repo]]
+url = "https://github.com/shumshersubashgautam/instasolanaweb3.0"
+
+[[repo]]
+url = "https://github.com/bilira-org/tender"
+
+[[repo]]
+url = "https://github.com/miclegoden/Taxi-betting-game-Node-express-Postgres-typescript-"
+
+[[repo]]
+url = "https://github.com/vking45/clusters"
+
+[[repo]]
+url = "https://github.com/kaizencorps/stache"
+
+[[repo]]
+url = "https://github.com/thirdweb-example/express-typescript-solana-starter"
+
+[[repo]]
+url = "https://github.com/scapula07/solana-payroll"
+
+[[repo]]
+url = "https://github.com/Web3-Builders-Alliance/Alberto-Q1-Solana-2023"
+
+[[repo]]
+url = "https://github.com/gumhq/sdk"
+
+[[repo]]
+url = "https://github.com/eqlabs/eclipse"
+
+[[repo]]
+url = "https://github.com/swaroopmaddu/solana-token-sale-program"
+
+[[repo]]
+url = "https://github.com/shumshersubashgautam/Tiktoksweb3solana"
+
+[[repo]]
+url = "https://github.com/pratik-mk/Decense"
+
+[[repo]]
+url = "https://github.com/kin-starters/kin-dapp-mogami"
+
+[[repo]]
+url = "https://github.com/ZclubWeb3/zclub-solana"
+
+[[repo]]
+url = "https://github.com/Mohamed6995/1st-project"
+
+[[repo]]
+url = "https://github.com/GoddessLuxanna/M2_Challenge_3-Solana-"
+
+[[repo]]
+url = "https://github.com/Unboxed-Software/solana-admin-instructions"
+
+[[repo]]
+url = "https://github.com/eyusufatik/learning-rust-and-solana"
+
+[[repo]]
+url = "https://github.com/thinkrapido/RoboSwapProgram"
+
+[[repo]]
+url = "https://github.com/Altar12/on-chain-tic-tac-toe-stake"
+
+[[repo]]
+url = "https://github.com/thirdweb-example/next-javascript-solana-starter"
```

Comment:
can you create an ecosystem for Thirdweb Templates, remove the repos in here that belong to it and add the ecosystem as a sub ecosytems to solana?

Reply:
Thirdweb does a lot more than Solana. Does it make sense to make it a subecosystem to Solana?

Reply:
Good observation. You can add it as a sub ecosystem of Solana and I will add it as a sub-ecosystem of the other compatible ecosystems later on